frontend/src/store/queries/index.ts

```diff
@@ -0,0 +1,23 @@
+import { ENV } from '@common/enums/app/env.enum';
+import {
+  createApi,
+  fetchBaseQuery,
+  BaseQueryFn,
+  FetchArgs,
+} from '@reduxjs/toolkit/query/react';
+
+type ErrorType = {
+  data: {
+    message: string;
+  };
+  status: number;
+};
+
+export const Api = createApi({
```

Comment:
We should use the names that start in uppercase. The `camelCase` is the standard